### PR TITLE
minor pipeline improvements

### PR DESCRIPTION
**Pull request by pivotal-jamie-klassen**
* track image used to run unit tests on PRs via resource
* no more ((vars)) directly in task configs

### CODE REVIEW

File: pipelines/prs.yml [hunk 1]
```diff
@@ -60,6 +60,11 @@ resources:
   icon: docker
   source: {repository: concourse/dev}
 
+- name: unit-image
```
Comment (cirocosta) [inline pipelines/prs.yml]: I was wondering the other day about renaming this image 🤔  At least for me, the name `unit` doesn't convey much what it really brings (a common set of tools that many tasks need).

Comment (pivotal-jamie-klassen) [reply pipelines/prs.yml]: yeah, I'd be happy to revisit the naming convention around `unit` and `dev`, but I have no bright ideas at this moment.

Comment (pivotal-jamie-klassen) [reply pipelines/prs.yml]: yeah, I'd be happy to revisit the naming convention around `unit` and `dev`, but I have no bright ideas at this moment.

Comment (kcmannem) [reply pipelines/prs.yml]: sameer and I were confused as well. We were thinking somthing like `test-tools` or `test-utils`

Comment (pivotal-jamie-klassen) [reply pipelines/prs.yml]: #118

**Pull request merged**